### DISCUSSION TITLE
Hubs-Cloud update in progress stuck

### DISCUSSION THREAD

**Discussion by elmau** [mozilla/hubs#3406] [Hubs Cloud Questions] [answered]
I'm trying to update the stack, but it gets stuck in the update and I can't do anything except cancel the update. What could be the problem?

**Comment by emclaren** [comment]
Hi there! Thanks for posting this in Github Discussions. Is the AWS console showing an error messages - can you send us a screenshot of what's happening?

**Reply by elmau** [reply]
It doesn't shows errors. it just rolls back.

**Comment by elmau** [comment, marked as answer]
I got this on the screen, it's strange because it updated to a higher ec2 instance, but then it rolled back to t3.medium
 
![Captura de pantalla 2020-11-20 152010](https://user-images.githubusercontent.com/14818268/99835873-02391d00-2b44-11eb-80fc-b46fe23c7102.png)

**Reply by robinkwilson** [reply]
Quick questions:

1. Is this an enterprise multi-server or personal/enterprise 1 server stack?
1. Are you updating the # of servers + the server size in this update?
1. I think that is the only error, but are there any more below?
1. Can you successfully increase the server size to a different size with no problems or is it only on an update to a 24xlarge servers?

**Reply by elmau** [reply]
It's an enterprise multi server, I have a t3.medium with 1x1 servers. And I want it to update to a 24xlarge server1x1, I could update it to a c5.4xlarge, but then it rolled back to t3 after a while.

**Reply by elmau** [reply]
I also managed at some point to make a 2x2 c5.large, however it had stability problems for the users. We got frozen streamings, problems with audio and such.

**Reply by robinkwilson** [reply]
Can you try restarting your EC2 instances:
AWS console > EC2 > click on your HC server(s) then "Terminate" instances. New ones will start up immediately after.

Then try the stack update ~5 mins after new one spins up.

**Reply by elmau** [reply]
ok, I'll try it. I'll let you know how it goes. I won't lose any data, right?

**Reply by elmau** [reply]
It worked, however I noticed that AWS provides a limited amount of VCPUs per bucket, and you have to request an increase on VCPUs in order to get a bigger stack. Once the approval was done I could upgrade to the required server without problems.

**Reply by robinkwilson** [reply]
Oh I haven't heard about that request in AWS. I'd appreciate any other information around that if you have it.

**Reply by elmau** [reply]
I don't have the screenshots for that, but when I tried to upscale without success a couple of times, in the EC2 console I saw a "Fail" state, where it says something about the current bucket having only a capacity of 32 vCPU, and an increase of vCPU should be requested.

**Reply by robinkwilson** [reply]
Thank you for that!
